src/components/select/Select.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import { colors } from 'common/colors';
+
+const Select = styled.select`
+  transition: all 0.15s cubic-bezier(0.645, 0.045, 0.355, 1);
```

Comment:
I'd prefer having transition set to the specific properties you want to transition, so we don't get weird side-effects. Also, is the cubic bezier really needed over the built-in keywords?